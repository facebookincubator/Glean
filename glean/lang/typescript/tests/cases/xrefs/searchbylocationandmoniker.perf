{
  "@generated": null,
  "lsif.DefinitionMoniker.1": 1,
  "lsif.MonikerDefinition.1": 1,
  "lsif.MonikerId.1": 1,
  "lsif.Name.1": 1,
  "src.File.1": 1
}