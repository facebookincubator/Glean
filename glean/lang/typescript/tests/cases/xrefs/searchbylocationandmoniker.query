query: lsif.SearchByLocationAndMoniker { file = "glean/lang/typescript/tests/cases/xrefs/example.ts", name = "createTempRepo", ident = "example:createTempRepo" }
perf: True
transform: [gensort, []]
