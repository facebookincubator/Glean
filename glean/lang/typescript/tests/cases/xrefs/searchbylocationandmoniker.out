[
  "@generated",
  {
    "key": {
      "file": { "key": "glean/lang/typescript/tests/cases/xrefs/example.ts" },
      "name": { "key": "createTempRepo" },
      "ident": { "key": "example:createTempRepo" },
      "entity": {
        "typescript": {
          "defn": {
            "key": {
              "defn": {
                "key": {
                  "file": {
                    "key": {
                      "file": {
                        "key": "glean/lang/typescript/tests/cases/xrefs/example.ts"
                      },
                      "language": 49
                    }
                  },
                  "range": {
                    "key": {
                      "range": {
                        "lineBegin": 46,
                        "columnBegin": 17,
                        "lineEnd": 46,
                        "columnEnd": 30
                      },
                      "fullRange": {
                        "lineBegin": 46,
                        "columnBegin": 1,
                        "lineEnd": 52,
                        "columnEnd": 1
                      },
                      "text": { "key": "createTempRepo" }
                    }
                  }
                }
              },
              "moniker": {
                "key": {
                  "kind": 0,
                  "scheme": { "key": "tsc" },
                  "ident": { "key": "example:createTempRepo" }
                }
              }
            }
          }
        }
      }
    }
  }
]